README.md

```diff
@@ -1,2 +1,27 @@
 # openstack_taster
 Tests a complete OpenStack deployment for various functionalities
+
+# Usage
+
+1. Make the following variables available through your shell:
+
+* **OS_USERNAME** -- Username to authenticate to the OpenStack API
+* **OS_PASSWORD** -- plain text password for the user account
+* OS_TENANT_NAME admin -- the "project" or tenant under which the test
+  instances will be created. Make sure you have permissions and resources to
+  create m1.small instances and one single free public ip.
+* **OS_AUTH_URL** -- URL where OpenStack API lives. something like https://openpower-openstack.testing.osuosl.org:5000/v2.0
+* **OS_REGION_NAME** -- The region where the instances should be created.
+* **OS_PRIVATE_SSH_KEY** -- Path to the private key of the pair to access these created
+  instances using ssh.
+* **OS_PUBLIC_SSH_KEY** -- Path to the public key of the keypair which will be put
+  in the instances once they are created
+* **OS_SSH_KEYPAIR** -- Name under which the public key is *already* saved in the
+  OpenStack Key Pairs.
```

Comment:
I don't have the FLOATING_IP_POOL and AVAILABILITY_ZONE and it works for me.

Fixing the formatting for OS_TENANT NAME.